scripts/qiita_test_install

```diff
@@ -0,0 +1,403 @@
+        ('bcrypt library version', bcrypt_lib_version),
+        ('pyparsing library version', pyparsing_lib_version),
+        ('networkX library version', networkx_lib_version),
+        ('WTForms library version', wtforms_lib_version),
+        ('qiime library version', qiime_lib_version + ', for more info use '
+         'print_qiime_config.py'),
+        ('mock library version', mock_lib_version),
+        ('MOI library version', moi_lib_version),
+        ('MOI config file path', environ['MOI_CONFIG_FP']),
+        ('IPython library version', ipython_lib_version),
+        ('psycopg2 library version', psycopg2_lib_version),
+        ('Qiita core library version', qiita_core_lib_version),
+        ('Qiita db library version', qiita_db_lib_version),
+        ('Qiita pet library version', qiita_pet_lib_version),
+        ('Qiita ware library version', qiita_ware_lib_version)
+    ]
+    max_len = max([len(e[0]) for e in version_info])
+    print dependency_info_header
+    for v in version_info:
+        print "%*s:\t%s" % (max_len, v[0], v[1])
+
+    extra_info = None
+    qiita_config = ConfigurationManager()
+    try:
+        qiita_conf_fp = environ['QIITA_CONFIG_FP']
+    except KeyError:
+        qiita_conf_fp = join(dirname(abspath(__file__)),
+                             'support_files/config_test.txt')
+    smtp = SMTP_SSL() if qiita_config.smtp_ssl else SMTP()
+    smtp.set_debuglevel(False)
+    try:
+        smtp.connect(qiita_config.smtp_host, qiita_config.smtp_port)
+        smtp.verify
+        send_email = True
+    except:
+        send_email = False
+    ebi_credentials = (qiita_config.ebi_access_key != '' and
+                       qiita_config.ebi_center_name != '' and
+                       qiita_config.ebi_dropbox_url != '' and
+                       qiita_config.ebi_organization_prefix != '' and
+                       qiita_config.ebi_seq_xfer_pass != '' and
+                       qiita_config.ebi_seq_xfer_url != '' and
+                       qiita_config.ebi_seq_xfer_user != '' and
+                       qiita_config.ebi_skip_curl_cert != '')
+    vamps_credentials = (qiita_config.vamps_pass != '' and
+                         qiita_config.vamps_url != '' and
+                         qiita_config.vamps_user != '')
+    try:
+        SQLConnectionHandler()
+        psql_running = True
+    except:
+        psql_running = False
+    try:
+        StrictRedis(
+            host=qiita_config.redis_host,
+            password=qiita_config.redis_password,
+            port=qiita_config.redis_port,
+            db=qiita_config.redis_db)
+        redis_running = True
+    except:
+        redis_running = False
+    try:
+        def sum(a, b):
+            return a, b
+
+        _, _, ar = submit_nouser(sum, 1, 2)
+        # wait up to 5 secs to hear back
+        ar.wait(timeout=5)
+        if ar.ready():
+            ipython_running = True
+        else:
+            ipython_running = False
+    except e:
```

Comment:
Line 365 is still thorwing a traceback and not showing the rest on my computer. Perhaps this should be 
```python
except Exception as e:
```
or just
```
except:
```
since we don't actually need e.